Sources/Interaction/Manipulators/MouseCameraUnicamRotateManipulator/index.js

```diff
@@ -86,18 +86,18 @@ function vtkMouseCameraUnicamRotateManipulator(publicAPI, model) {
     cameraFocalPoint[3] = 1.0;
     cameraViewUp[3] = 0.0;
 
-    const transform = mat4.create();
+    const transform = new Float64Array(16);
     mat4.identity(transform);
-    mat4.translate(transform, transform, vec3.fromValues(cx, cy, cz));
-    mat4.rotate(transform, transform, angle, vec3.fromValues(ax, ay, az));
-    mat4.translate(transform, transform, vec3.fromValues(-cx, -cy, -cz));
+    mat4.translate(transform, transform, new Float64Array([cx, cy, cz]));
+    mat4.rotate(transform, transform, angle, new Float64Array([ax, ay, az]));
+    mat4.translate(transform, transform, new Float64Array([-cx, -cy, -cz]));
```

Comment:
When you do operation like that, you don't need to rework the array. You can leave [-cx, -cy, -cz] directly. That skip 1 new. 

If you want to prevent any new you could have a `tmp3 = Float64Array(3)` and call `vec3.set(tmp3, -cx, -cy, -cz)`

Reply:
Yes I have already edited that locally. I pushed this so I could take a look at it as I find it easier to see the changes here.